### PR TITLE
Fix bash completions and init/systemd user integrations

### PR DESCRIPTION
**Pull request by pavinjosdev**
Hi Luca,

This PR fixes the following:
1. Bash completions for the `distrobox` wrapper command
2. Auth cookie symlink in user integration for systemd/init containers
3. User integrations from being cleaned up by systemd when no session for the user is active, which causes future sessions to break (no access to wayland/pipewire, etc).

🚀

### CODE REVIEW

File: distrobox-init [hunk 1]
```diff
@@ -2210,6 +2210,7 @@ mkdir -p /run/user/\$(id -ru)/keyring && ln -sf /run/host/run/user/\$(id -ru)/ke
 mkdir -p /run/user/\$(id -ru)/p11-kit && ln -sf /run/host/run/user/\$(id -ru)/p11-kit/* /run/user/\$(id -ru)/p11-kit/
 mkdir -p /run/user/\$(id -ru)/pulse && ln -sf /run/host/run/user/\$(id -ru)/pulse/* /run/user/\$(id -ru)/pulse/
 find /run/user/\$(id -ru) -maxdepth 2 -xtype l -delete
+loginctl enable-linger \$(id -ru)
```
[the remark this answers is not in the document]
Comment (89luca89) [reply distrobox-init:2213]: `registry.opensuse.org/opensuse/distrobox:latest` on Aeon host (so similar to TW)

[the remark this answers is not in the document]
Comment (pavinjosdev) [reply distrobox-init:2213]: @89luca89 Thanks, I also found out the same issue in a new debian container with init.
Working on a solution! 🚀

[the remark this answers is not in the document]
Comment (pavinjosdev) [reply distrobox-init]: @89luca89 It's fixed now. Tested against new debian 12 and opensuse init containers.

[the remark this answers is not in the document]
Comment (89luca89) [reply distrobox-init]: Thanks!

**Review by 89luca89 — APPROVED** [review]

**Comment by 89luca89** [comment]
Thanks a lot @pavinjosdev !

**Pull request merged**